Demo/SPGooglePlacesAutocompleteDemoTests/SPGooglePlacesAutocompleteDemoTests.m

```diff
@@ -0,0 +1,176 @@
+                                                     @{
+                                                         @"offset" : [[NSNumber alloc] initWithInt:7],
+                                                         @"value" : @"France",
+                                                         }],
+                                               @"types" :
+                                                   @[@"locality", @"political", @"geocode"],
+                                               },
+                                           @{
+                                               @"description" : @"Paris, France",
+                                               @"id" : @"691b237b0322f28988f3ce03e321ff72a12167fd",
+                                               @"matched_substrings" :
+                                                   @{
+                                                       @"length" : [[NSNumber alloc] initWithInt:2],
+                                                       @"offset" : [[NSNumber alloc] initWithInt:0]
+                                                       },
+                                               @"place_id" : @"ChIJD7fiBh9u5kcRYJSMaMOCCwQ",
+                                               @"reference" : @"ClRPAAAAOV6o5AANjB6uMv1YPsyTyOliJvOW0kkm_jBOUiEDgzK7lmn1hvOVY0mZQzoV7nC5OcU8aKPhTRlmMDHFg1FUklX7b5AqPcowTx8qNYqnNAMSEDr56BSrORjB1fhKFBAKsuEaFAZfh-DqFAlnvI20BYNWnQ2mfldO",
+                                               @"terms" :
+                                                   @[@{
+                                                         @"offset" : [[NSNumber alloc] initWithInt:0],
+                                                         @"value" : @"Paris",
+                                                         },
+                                                     @{
+                                                         @"offset" : [[NSNumber alloc] initWithInt:7],
+                                                         @"value" : @"France",
+                                                         }],
+                                               @"types" :
+                                                   @[@"locality", @"political", @"geocode"],
+                                               }
+                                           ],
+                                   @"status" : @"OK"
+    };
+        return [OHHTTPStubsResponse responseWithJSONObject:response
+                                                statusCode:200 headers:nil];
+    }];
+
+
+    self.qry.input = @"Paris";
+    XCTestExpectation *expectation = [self expectationWithDescription:@"PlacesSearchs"];
+    [self.qry fetchPlaces:^(NSArray *places, NSError *error) {
+        XCTAssertEqual([places count], 2);
+        XCTAssertNil(error);
+        [expectation fulfill];
+    }];
+    [self waitForExpectationsWithTimeout:40.0 handler:nil];
```

Comment:
You could be a little more agressive on the timeout value, 40 seconds seems to be a lot.